docker-compose.prod.yml

```diff
@@ -0,0 +1 @@
+name: bailo-prod
```

Comment:
I'm not sure if you've intentionally renamed this file or not, but some of the builds are failing because they're looking for `docker-compose-prod.yml` not `docker-compose.prod.yml`